app/steps/ui/summary/includes/executors-applicant.njk

```diff
@@ -16,10 +16,20 @@
     alreadyDeclared = fields.summary.alreadyDeclared.value
 )}}
 
-{% set applicantNameAsOnWillQuestion = content.ApplicantNameAsOnWill.questionWithoutName %}
+{# this seems like the worst possible way to have a boolean comparison #}
+{# we're passing a boolean into this from js and we end up with a string? i have no idea here #}
```

Comment:
Is this something we can expand on through a tech debt ticket?

Reply:
It would seem reasonable, but I don't know whether we will ever get to this